includes/general.php

```diff
@@ -684,6 +684,23 @@ function pods_access( $privs, $method = 'OR' ) {
  * @since 1.6.7
  */
 function pods_shortcode( $tags, $content = null ) {
+	try {
+		return pods_shortcode_run( $tags, $content );
+	} catch ( Exception $exception ) {
+		return $exception->getMessage();
+	}
+}
+
+/**
+ * Shortcode support for use anywhere that support WP Shortcodes
+ *
+ * @param array  $tags    An associative array of shortcode properties
+ * @param string $content A string that represents a template override
+ *
+ * @return string
+ * @since 2.7.13
+ */
+function pods_shortcode_run( $tags, $content = null ) {
```

Comment:
@sc0ttkclark  I assume this is a private function? Maybe prepend it with an underscore?

Reply:
Not a fan of underscores for private functions, personally, it's a convention that isn't always clear and doesn't enforce anything.  Comments in the function's header is a better idea for cases like this where it can't just be made private, IMO.  Just my $.02, not a huge deal either way.

Reply:
We'll refactor this in the future when we expand shortcodes/blocks more. Not anything to worry about right now though.